### DISCUSSION TITLE
100l Analogzeiger dreht Ehrenrunde

### DISCUSSION THREAD

**Discussion by Esp2use** [jomjol/AI-on-the-edge-device#1683] [Q&A]
Hallo,
so wie es ausschaut dreht mein 100l-Anlogzeiger (x0,1) ne Ehrenrunde bevor 1m³ vollständig erreicht ist. Der Wasseruhraustausch war erst letztes Jahr, so dass der nächste Wechsel noch ein wenig dauert. Meine Frage geht nun dahin, ob es eine Möglichkeit gibt, dies irgendwie abzufangen, oder ob hier ein aktivierbarer Teiler implementiert werden kann. Ich habe jetzt keinen Überblick, ob so etwas öfter vor kommt, dass die Uhren Kundenfreundlich funktionieren ;-) Bin für hilfreiche Tipps dankbar. Ansonsten bin ich von dieser Software begeistert. 
![wasseruhr_ehrenrunde](https://user-images.githubusercontent.com/121380199/209478609-84f87443-16d8-49b1-a681-1f525f7fe210.PNG)
Danke und Gruß

**Comment by Esp2use** [comment]
@jomjol 
Hallo,
wäre eine andere Teilung, anstatt 0,1 für die 100Liter-Anzeige, 0,05, implementierbar? Ggf. versteckt i.d. ini? 
Viele Grüße

**Comment by jomjol** [comment]
Deine "defekte" Wasseruhr scheint mir ein ziemlicher Sonderfall zu sein.
Anpassung des Postprocessings ist eher aufwendig und Änderungen sehr fehleranfällig. Da will ich keine Sonderlösung für so etwas einbauen - zumal es vermutlich ein ziemlicher Einzelfall ist.

**Reply by Esp2use** [reply]
Ok, danke für die Info.
